File: JAVA/Count the elements.java
class Solution
{
    public static int[] countElements(int a[], int b[], int n, int query[], int q)
    {
        int res[] = new int[q];
        int ind = 0;
        for(int i: query) 
        {
            int ele = a[i], count = 0;
            for(int num: b)
                if(num <= ele)
                    count++;
            res[ind++] = count;
        }
        return res;
    }
}

File: C++/Count the elements.cpp
class Solution {
public:
    vector<int> countElements(vector<int> &a, vector<int> &b, int n, vector<int> &query, int q) {
        vector<int> ans;
        sort(b.begin(), b.end());
        for(auto i: query) {
            int ele = a[i], count = upper_bound(b.begin(), b.end(), ele) - b.begin();
            ans.push_back(count);
        }
        return ans;
    }
};
